packages/core/app/styles/navi-core/components/navi-visualization-config/series-chart.less

```diff
@@ -0,0 +1,10 @@
+/**	
+ * Copyright 2019, Yahoo Holdings Inc.	
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.	
+ */
+
+.metric-select {
+  &__select-trigger {
+    flex: 1;
+  }
```

Comment:
Going forward we should use `.scss` files and add `index.scss` and imports as needed up to the root `.scss` file, but we can probably just remove this style and use the `w-full` helper for the `@triggerClass` of the power select

Reply:
Oh you're right I forgot that denali has helpers for flex box as well, I'll re-delete the file and add the helper on instead.